v2/pkg/cli/executor.go

```diff
@@ -450,27 +451,30 @@ http:
       #path: /etc/registry
 `
 
-	if _, err := os.Stat(o.LocalStorageDisk); err != nil {
-		// something went wrong
-		return fmt.Errorf("error using the local storage folder for caching")
-	}
-	configYamlV01 = strings.Replace(configYamlV01, "$$PLACEHOLDER_ROOT$$", o.LocalStorageDisk, 1)
-	configYamlV01 = strings.Replace(configYamlV01, "$$PLACEHOLDER_PORT$$", strconv.Itoa(int(o.Opts.Global.Port)), 1)
-	if o.Opts.Global.LogLevel != "trace" {
-		configYamlV01 = strings.Replace(configYamlV01, "$$PLACEHOLDER_LOG_LEVEL$$", o.Opts.Global.LogLevel, 1)
-	} else {
-		configYamlV01 = strings.Replace(configYamlV01, "$$PLACEHOLDER_LOG_LEVEL$$", "info", 1)
```

Comment:
trace is not a log level understood , so you need some sort of translation for it. that is why, we cannot simply use: 
```
rc := RegistryConfig{
		LocalStorageDisk: o.LocalStorageDisk,
		LocalStoragePort: int(o.Opts.Global.Port),
		LogLevel:         o.Opts.Global.LogLevel,
		LogAccessOff:     o.Opts.Global.LogLevel != "debug",
	}
```